non-atomic-action-interpreter-module/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
We use MIT LICENSE for OSTIS project. Why do you use GNU?

Reply:
Not all OSTIS-AI repositories use the MIT license. For example, scl-machine uses the Apache license. As for this repository, this license was chosen before me. There may have been reasons for that. Let's discuss